web/src/components/molecules/Integration/types.ts

```diff
@@ -6,7 +6,7 @@ export type IntegrationMember = {
   active: boolean;
 };
 
-export type Role = "WRITER" | "READER" | "OWNER";
+export type Role = "WRITER" | "READER" | "OWNER" | "Maintainer";
```

Comment:
I think I saw earlier you discussing this, but is it correct that it is `Maintainer` and not `MAINTAINER` because of a mistake on the backend? If so, please resolve this message

Reply:
Yes this is because of the backend.